src/components/AuthPage/index.js

```diff
@@ -30,14 +30,14 @@ const AuthPage = ({ type }) => {
       justify="center"
       className="row-footer-below auth-margin"
     >
-      <Grid item={true} xs={12} sm={12} md={8} lg={5} className="auth-form-col">
+      <Grid item={true}>
         <Fade
-          left={isDesktop ? showType === "login" : false}
-          right={
-            isDesktop
-              ? showType === "signup" || showType === "forgotpassword"
-              : false
-          }
+          // left={isDesktop ? showType === "login" : false}
+          // right={
+          //   isDesktop
+          //     ? showType === "signup" || showType === "forgotpassword"
+          //     : false
+          // }
```

Comment:
I commented this because with our new login page I have added our own animation effect when the login screen will appear , Thats why I removed the animation from the <Fade/> component. I will remove this comment line in next commit